crates/wasmtime/src/externals.rs

```diff
@@ -23,7 +24,7 @@ use wasmtime_runtime::{self as runtime, InstanceHandle};
 #[derive(Clone)]
 pub enum Extern {
     /// A WebAssembly `func` which can be called.
-    Func(Func),
+    Func(Option<Func>),
```

Comment:
The fallout from this change seems quite large (affecting the C API and many callers of the default API), but I can't figure out why this changed? 

While this makes sense for `Val` I don't think this makes sense for `Extern` which is acquired from an instance, and I don't think that instances can export a `null` funcref, right?

Reply:
I... *think* you are correct. I'll look into it.